extensions-core/s3-extensions/src/main/java/io/druid/storage/s3/S3StorageDruidModule.java

```diff
@@ -98,16 +101,9 @@ public void configure(Binder binder)
 
   @Provides
   @LazySingleton
-  public AWSCredentialsProvider getAWSCredentialsProvider(final AWSCredentialsConfig config)
```

Comment:
Oh, I think I see your question. Yes, from `AWSModule`

The problem was that the s3 extension overrode the `AWSModule` with the exact same code!